### ISSUE TITLE
test_norm_3d_inner_axis blocks landing torch.norm improvements on pytorch/pytorch

### ISSUE THREAD

**Issue by kit1980** [pytorch/glow#5994]
This PR https://github.com/pytorch/pytorch/pull/81761 has several improvements for `torch.norm`, but can't be landed because it fails a glow test:

```
FAIL: test_norm_3d_inner_axis (glow.torch_glow.nodes.norm_test.TestNorm)
Basic test of the PyTorch norm Node on Glow.

AssertionError: Expected fusion of {'aten::frobenius_norm'}, but only set() was fused in graph
graph(%self : __torch__.glow.torch_glow.nodes.norm_test.SimpleNormModule,
      %tensor : Float(*, *, *, requires_grad=0, device=cpu)):
  %8 : int[] = prim::Constant[value=[1]]()
  %2 : NoneType = prim::Constant()
  %3 : bool = prim::Constant[value=0]() # /data/sandcastle/boxes/eden-trunk-hg-fbcode-fbsource/fbcode/buck-out/dev/gen/aab7ed39/glow/glow/torch_glow/tests/norm_test#binary,link-tree/torch/functional.py:1471:0
  %5 : int = prim::Constant[value=2]() # /data/sandcastle/boxes/eden-trunk-hg-fbcode-fbsource/fbcode/buck-out/dev/gen/aab7ed39/glow/glow/torch_glow/tests/norm_test#binary,link-tree/torch/functional.py:1471:0
  %7 : Tensor = aten::linalg_vector_norm(%tensor, %5, %8, %3, %2) # /data/sandcastle/boxes/eden-trunk-hg-fbcode-fbsource/fbcode/buck-out/dev/gen/aab7ed39/glow/glow/torch_glow/tests/norm_test#binary,link-tree/torch/functional.py:1471:0
  return (%7)
```

See https://github.com/pytorch/pytorch/pull/81761 for more details.

Any suggestions how to unblock the PR?

**Comment by malfet** [comment]
@jfix71 do you know the answer by any chance?

**Comment by opti-mix** [comment]
cc @jackm321 and @jfix71 Do you have any insights regarding this? Thanks!

**Comment by jfix71** [comment]
Seems like the original PR is now dispatching `torch.norm` to `linalg.vector_norm` instead of `frobenius_norm`, so we need to update the loader to load `linalg.vector_norm` as seen below? CC @khabinov @qxy11

https://github.com/pytorch/glow/blob/87753a10fa3d99554e6f50119996917c33d63f4b/torch_glow/src/PyTorchModelLoader.cpp#L1721-L1723

**Comment by qxy11** [comment]
Yes, the loader has to be updated to also load `aten::linalg_vector_norm` at
https://github.com/pytorch/glow/blob/87753a10fa3d99554e6f50119996917c33d63f4b/torch_glow/src/PyTorchModelLoader.cpp#L1721-L1723

Also this test case needs to be updated to `aten::linalg_vector_norm` instead of `aten::frobenius_norm`
https://github.com/pytorch/glow/blob/1b4e9d1fdcb0fc8abf7a6be26a781256bc307b17/torch_glow/tests/nodes/norm_test.py#L56

**Comment by kit1980** [comment]
Can we disable the test for now and update it after the PyTorch change lands?

**Comment by qxy11** [comment]
Yes but please update the section below with `{{"aten::norm", "aten::frobenius_norm", "aten::linalg_vector_norm"},`(and it'd be great if you could try those unit tests locally to make sure that change works). If torch::norm is used in any models running on A*, the PR might break model loading without adding this change. 
https://github.com/pytorch/glow/blob/87753a10fa3d99554e6f50119996917c33d63f4b/torch_glow/src/PyTorchModelLoader.cpp#L1721-L1723

**Comment by Lezcano** [comment]
What's the current state of this issue?

**Comment by kit1980** [comment]
@qxy11 I have a PR, please take a look: https://github.com/pytorch/glow/pull/6003
Unfortunately I was not able to test it locally as the compilation setup for glow is a bit complicated.
